// @haystacks/constants/src/constants/color.constants.js
/**
 * @file color.constants.js
 * @module color.constants
 * @description Contains all named colors.
 * @requires module:basic.constants
 * @requires module:phonic.constants
 * @requires module:generic.constants
 * @requires module:numeric.constants
 * @requires module:word.constants
 * @requires module:language.constants
 * @requires module:country.constants
 * @author Seth Hollingsead
 * @date 2022/02/21
 * @copyright Copyright © 2021-… by Seth Hollingsead. All rights reserved
 */

import * as bas from './basic.constants.js';
import * as phn from './phonic.constants.js';
import * as gen from './generic.constants.js';
import * as num from './numeric.constants.js';
import * as wrd from './word.constants.js';
import * as lng from './language.constants.js';
import * as ctr from './country.constants.js';

// NOTE: I changed e-x-p-o-r-t c-o-n-s-t => smuggle something,
// because the constants validation system is scanning these files looking for that keyword and
// if it finds it then it will use that line for vaidation purposes.
// So to reverse this, just do a find replace and chane // s.muggle soemthign back into the other e-x-p-o-r-t c-o-n-s-t string.
// The current state of the application doesn't need all of these, but future applications will most definately need them.
// So they are left here as a matter of future-proofing.

// Colors
// Primary Colors
export const cred = bas.cr + bas.ced; // red
export const cRed = bas.cR + bas.ced; // Red
export const cgreen = bas.cg + phn.creen; // green
export const cGreen = bas.cG + phn.creen; // Green
export const cblue = bas.cb + phn.clue; // blue
export const cBlue = bas.cB + phn.clue; // Blue
// Secondary Colors CMYK
export const ccyan = bas.cc + phn.cyan; // cyan
export const cCyan = bas.cC + phn.cyan; // Cyan
export const cmagenta = bas.cm + bas.cag + phn.centa; // magenta
export const cMagenta = bas.cM + bas.cag + phn.centa; // Magenta
export const cyellow = wrd.cyell + bas.cow; // yellow
export const cYellow = wrd.cYell + bas.cow; // Yellow
export const cblack = bas.cb + wrd.clack; // black
export const cBlack = bas.cB + wrd.clack; // Black
export const cwhite = bas.cwh + phn.cite; // white
export const cWhite = bas.cWh + phn.cite; // White
export const corange = bas.cor + phn.cange; // orange
export const cOrange = bas.cOr + phn.cange; // Orange

// Common Colors
export const cbrown = bas.cbr + phn.cown; // brown
export const cBrown = bas.cBr + phn.cown; // Brown
export const cgrey = bas.cg + phn.crey; // grey
export const cGrey = bas.cG + phn.crey; // Grey
export const cgray = bas.cg + phn.cray; // gray
export const cGray = bas.cG + phn.cray; // Gray
export const cpink = bas.cp + phn.cink; // pink
export const cPink = bas.cP + phn.cink; // Pink
export const clavender = bas.cl + phn.cavender; // lavender
export const cLavender = bas.cL + phn.cavender; // Lavender
export const cpurple = bas.cp + phn.curple; // purple
export const cPurple = bas.cP + phn.curple; // Purple
export const cviolet = bas.cv + phn.ciolet; // violet
export const cViolet = bas.cV + phn.ciolet; // Violet
export const clilac = bas.cl + phn.cilac; // lilac
export const cLilac = bas.cL + phn.cilac; // Lilac
export const cmaroon = bas.cm + phn.caroon; // maroon
export const cMaroon = bas.cM + phn.caroon; // Maroon
export const crose = bas.cr + phn.cose; // rose
export const cRose = bas.cR + phn.cose; // Rose
export const cscarlet = bas.csc + bas.car + wrd.clet; // scarlet
export const cScarlet = bas.cSc + bas.car + wrd.clet; // Scarlet
export const cruby = bas.cr + phn.cuby; // ruby
export const cRuby = bas.cR + phn.cuby; // Ruby
export const cgold = bas.cg + wrd.cold; // gold
export const cGold = bas.cG + wrd.cold; // Gold
export const csilver = bas.cs + phn.cilver; // silver
export const cSilver = bas.cS + phn.cilver; // Silver
export const ccopper = bas.cc + phn.copper; // copper
export const cCopper = bas.cC + phn.copper; // Copper
export const ccobalt = bas.cc + phn.cobalt; // cobalt
export const cCobalt = bas.cC + phn.cobalt; // Cobalt
export const ctan = bas.cta + bas.cn; // tan
export const cTan = bas.cTa + bas.cn; // Tan
export const ctitanium = bas.cti + ctan + phn.cium; // titanium
export const cTitanium = bas.cTi + ctan + phn.cium; // Titanium
export const ccadmium = bas.cca + bas.cd + phn.cmium; // cadmium
export const cCadmium = bas.cCa + bas.cd + phn.cmium; // Cadmium
export const cplatnium = bas.cpl + bas.cat + phn.cnium; // platnium
export const cPlatnium = bas.cPl + bas.cat + phn.cnium; // Platnium
export const csapphire = wrd.csap + phn.cphire; // sapphire
export const cSapphire = wrd.cSap + phn.cphire; // Sapphire
export const cteal = bas.ct + phn.ceal; // teal
export const cTeal = bas.cT + phn.ceal; // Teal
export const ctaupe = bas.ct + phn.caupe; // taupe
export const cTaupe = bas.cT + phn.caupe; // Taupe
export const cdrab = bas.cd + phn.crab; // drab
export const cDrab = bas.cD + phn.crab; // Drab
export const cturquoise = bas.ctu + phn.crquoise; // turquoise
export const cTurquoise = bas.cTu + phn.crquoise; // Turquoise
export const ctangerine = ctan + phn.cgerine; // tangerine
export const cTangerine = cTan + phn.cgerine; // Tangerine
export const cumber = bas.cu + phn.cmber; // umber
export const cUmber = bas.cU + phn.cmber; // Umber
export const cvanilla = wrd.cvan + phn.cilla; // vanilla
export const cVanilla = wrd.cVan + phn.cilla; // Vanilla
export const caero = bas.ca + phn.cero; // aero
export const cAero = bas.cA + phn.cero; // Aero
export const caqua = bas.ca + phn.cqua; // aqua
export const cAqua = bas.cA + phn.cqua; // Aqua
export const camber = bas.ca + phn.cmber; // amber
export const cAmber = bas.cA + phn.cmber; // Amber
export const camethyst = bas.cam + phn.ceth + phn.cyst; // amethyst
export const cAmethyst = bas.cAm + phn.ceth + phn.cyst; // Amethyst
export const camaranth = bas.ca + phn.cmaranth; // amaranth
export const cAmaranth = bas.cA + phn.cmaranth; // Amaranth
export const cazure = bas.caz + phn.cure; // azure
export const cAzure = bas.cAz + phn.cure; // Azure
export const cbeige = bas.cbe + phn.cige; // beige
export const cBeige = bas.cBe + phn.cige; // Beige
export const ccherry = bas.cch + phn.cerry; // cherry
export const cCherry = bas.cCh + phn.cerry; // Cherry
export const ccrimson = bas.cc + phn.crimson; // crimson
export const cCrimson = bas.cC + phn.crimson; // Crimson
export const colive = bas.co + phn.clive; // olive
export const cOlive = bas.cO + phn.clive; // Olive
export const csienna = bas.cs + phn.cienna; // sienna
export const cSienna = bas.cS + phn.cienna; // Sienna
export const cfandango = bas.cf + phn.candango; // fandango
export const cFandango = bas.cF + phn.candango; // Fandango
export const cfuchsia = bas.cf + phn.cuchsia; // fuchsia
export const cFuchsia = bas.cF + phn.cuchsia; // Fuchsia
export const clime = bas.cl + phn.cime; // lime
export const cLime = bas.cL + phn.cime; // Lime
export const cindigo = bas.ci + phn.cndigo; // indigo
export const cIndigo = bas.cI + phn.cndigo; // Indigo
export const clemon = bas.cl + phn.cemon; // lemon
export const cLemon = bas.cL + phn.cemon; // Lemon
export const cmauve = bas.cm + phn.cauve; // mauve
export const cMauve = bas.cM + phn.cauve; // Mauve
export const cpeach = bas.cp + phn.ceach; // peach
export const cPeach = bas.cP + phn.ceach; // Peach
export const cplum = bas.cp + phn.clum; // plum
export const cPlum = bas.cP + phn.clum; // Plum
export const ctuscan = bas.ct + phn.cuscan; // tuscan
export const cTuscan = bas.cT + phn.cuscan; // Tuscan
export const cviridian = bas.cv + phn.ciridian; // viridian
export const cViridian = bas.cV + phn.ciridian; // Viridian
export const cvermilion = bas.cv + phn.cermilion; // vermilion
export const cVermilion = bas.cV + phn.cermilion; // Vermilion

// Tertiary Named Colors
// https://en.wikipedia.org/wiki/List_of_colors:_A%E2%80%93F
// https://en.wikipedia.org/wiki/List_of_colors:_G%E2%80%93M
// https://en.wikipedia.org/wiki/List_of_colors:_N%E2%80%93Z
export const cAbsoluteZero = wrd.cAbsolute + num.cZero; // AbsoluteZero
export const cAcidGreen = wrd.cAcid + cGreen; // AcidGreen
export const cAeroBlue = cAero + cBlue; // AeroBlue
export const cAfricanViolet = wrd.cAfrican + cViolet; // AfricanViolet
export const cAirSuperiorityBlue = wrd.cAir + wrd.cSuperiority + cBlue; // AirSuperiorityBlue
export const cAliceBlue = wrd.cAlice + cBlue; // AliceBlue
export const cAlloyOrange = wrd.cAlloy + cOrange; // AlloyOrange
export const cAlmond = bas.cAl + phn.cmond; // Almond
export const cAmaranthMP = cAmaranth + bas.cMP; // AmaranthMP
export const cAmaranthPink = cAmaranth + cPink; // AmaranthPink
export const cAmaranthPurple = cAmaranth + cPurple; // AmaranthPurple
export const cAmaranthRed = cAmaranth + cRed; // AmaranthRed
export const cAmazon = bas.cAm + bas.caz + bas.con; // Amazon
export const cAmberSAEECE = cAmber + bas.cSA + bas.cEE + bas.cCE; // AmberSAEECE
export const cAndroidGreen = wrd.cAndroid + cGreen; // AndroidGreen
export const cAntiqueBrass = wrd.cAntique + wrd.cBrass; // AntiqueBrass
export const cAntiqueBronze = wrd.cAntique + wrd.cBronze; // AntiqueBronze
export const cAntiqueFuchsia = wrd.cAntique + cFuchsia; // AntiqueFuchsia
export const cAntiqueRuby = wrd.cAntique + cRuby; // AntiqueRuby
export const cAntiqueWhite = wrd.cAntique + cWhite; // AntiqueWhite
export const cAoEnglish = bas.cAo + lng.cEnglish; // AoEnglish
export const cAppleGreen = wrd.cApple + cGreen; // AppleGreen
export const cAquamarine = cAqua + phn.cmar + phn.cine; // Aquamarine
export const cArcticLime = wrd.cArctic + cLime; // ArcticLime
export const cArmyGreen = wrd.cArmy + cGreen; // ArmyGreen
export const cArtichoke = bas.cAr + phn.ctic + bas.cho + bas.cke; // Artichoke
export const cArylideYellow = wrd.cArylide + cYellow; // ArylideYellow
export const cAshGray = wrd.cAsh + cGray; // AshGray
export const cAsparagus = bas.cAs + phn.cpara + bas.cgu + bas.cs; // Asparagus
export const cAtomicTangerine = wrd.cAtomic + cTangerine; // AtomicTangerine
export const cAuburn = bas.cAu + bas.cbu + bas.crn; // Auburn
export const cAureolin = bas.cAu + bas.cre + phn.colin; // Aureolin
export const cAvocado = bas.cAv + bas.coc + phn.cado; // Avocado
export const cAzureWeb = cAzure + wrd.cWeb; // AzureWeb
export const cBabyBlue = wrd.cBaby + cBlue; // BabyBlue
export const cBabyBlueEyes = wrd.cBaby + cBlue + wrd.cEyes; // BabyBlueEyes
export const cBabyPink = wrd.cBaby + cPink; // BabyPink
export const cBabyPowder = wrd.cBaby + wrd.cPowder; // BabyPowder
export const cBakerMillerPink = wrd.cBaker + wrd.cMiller + cPink; // BakerMillerPink
export const cBananaMania = wrd.cBanana + wrd.cMania; // BananaMania
export const cBarbiePink = wrd.cBarbie + cPink; // BarbiePink
export const cBarnRed = wrd.cBarn + cRed; // BarnRed
export const cBattleshipGrey = wrd.cBattleship + cGrey; // BattleshipGrey
export const cBeauBlue = wrd.cBeau + cBlue; // BeauBlue
export const cBeaver = bas.cBe + phn.cave + bas.cr; // Beaver
export const cBDazzledBlue = bas.cB + wrd.cDazzled + cBlue; // BDazzledBlue
export const cBigDipORuby = wrd.cBig + bas.cDi + bas.cpO + cRuby; // BigDipORuby
export const cBisque = bas.cBi + bas.csq + bas.cue; // Bisque
export const cBistreBrown = wrd.cBistre + cBrown; // BistreBrown
export const cBitterLemon = wrd.cBitter + cLemon; // BitterLemon
export const cBitterLime = wrd.cBitter + cLime; // BitterLime
export const cBittersweet = wrd.cBittersweet; // Bittersweet
export const cBittersweetShimmer = wrd.cBittersweet + wrd.cShimmer; // BittersweetShimmer
export const cBlackBean = cBlack + wrd.cBean; // BlackBean
export const cBlackChocolate = cBlack + wrd.cChocolate; // BlackChocolate
export const cBlackCoffee = cBlack + wrd.cCoffee; // BlackCoffee
export const cBlackCoral = cBlack + wrd.cCoral; // BlackCoral
export const cBlackOlive = cBlack + cOlive; // BlackOlive
export const cBlackShadows = cBlack + wrd.cShadows; // BlackShadows
export const cBlanchedAlmond = wrd.cBlanched + bas.cAl + phn.cmond; // BlanchedAlmond
export const cBlastOffBronze = wrd.cBlast + gen.cOff + wrd.cBronze; // BlastOffBronze
export const cBleuDeFrance = bas.cBl + bas.ceu + bas.cDe + ctr.cFrance; // BleuDeFrance
export const cBlizzardBlue = wrd.cBlizzard + cBlue; // BlizzardBlue
export const cBlond = bas.cBl + phn.cond; // Blond
export const cBloodRed = wrd.cBlood + cRed; // BloodRed
export const cBlueCrayola = cBlue + wrd.cCrayola; // BlueCrayola
export const cBlueMunsell = cBlue + wrd.cMunsell; // BlueMunsell
export const cBlueNCS = cBlue + phn.cNCS; // BlueNCS
export const cBluePantone = cBlue + wrd.cPantone; // BluePantone
export const cBluePigment = cBlue + wrd.cPigment; // BluePigment
export const cBlueRYB = cBlue + phn.cRYB; // BlueRYB
export const cBlueBell = cBlue + wrd.cBell; // BlueBell
export const cBlueGray = cBlue + cGray; // BlueGray
export const cBlueGreen = cBlue + cGreen; // BlueGreen
export const cBlueGreenColorWheel = cBlue + cGreen + wrd.cColor + wrd.cWheel; // BlueGreenColorWheel
export const cBlueJeans = cBlue + wrd.cJeans; // BlueJeans
export const cBlueSapphire = cBlue + cSapphire; // BlueSapphire
export const cBlueViolet = cBlue + cViolet; // BlueViolet
export const cBlueVioletCrayola = cBlue + cViolet + wrd.cCrayola; // BlueVioletCrayola
export const cBlueVioletColorWheel = cBlue + cViolet + wrd.cColor + wrd.cWheel; // BlueVioletColorWheel
export const cBlueYonder = cBlue + wrd.cYonder; // BlueYonder
export const cBluetiful = cBlue + bas.cti + phn.cful; // Bluetiful
export const cBlush = wrd.cBlush; // Blush
export const cBole = bas.cBo + bas.cle; // Bole
export const cBone = bas.cBo + bas.cne; // Bone
export const cBottleGreen = wrd.cBottle + cGreen; // BottleGreen
export const cBrandy = phn.cBra + phn.cndy; // Brandy
export const cBrickRed = wrd.cBrick + cRed; // BrickRed
export const cBrightGreen = wrd.cBright + cGreen; // BrightGreen
export const cBrightLilac = wrd.cBright + cLilac; // BrightLilac
export const cBrightMaroon = wrd.cBright + cMaroon; // BrightMaroon
export const cBrightNavyBlue = wrd.cBright + wrd.cNavy + cBlue; // BrightNavyBlue
export const cBrightYellowCrayola = wrd.cBright + cYellow + wrd.cCrayola; // BrightYellowCrayola
export const cBrilliantRose = wrd.cBrilliant + cRose; // BrilliantRose
export const cBrinkPink = bas.cBr + phn.cink + cPink; // BrinkPink
export const cBritishRacingGreen = bas.cBr + phn.citi + bas.csh + wrd.cRacing + cGreen; // BritishRacingGreen
export const cBronze = wrd.cBronze; // Bronze
export const cBrownSugar = cBrown + wrd.cSugar; // BrownSugar
export const cBrunswickGreen = wrd.cBrunswick + cGreen; // BrunswickGreen
export const cBudGreen = wrd.cBud + cGreen; // BudGreen
export const cBuff = bas.cBu + bas.cff; // Buff
export const cBurgundy = bas.cBu + phn.crgundy; // Burgundy
export const cBurlywood = wrd.cBurly + wrd.cwood; // Burlywood
export const cBurnishedBrown = wrd.cBurnished + cBrown; // BurnishedBrown
export const cBurntOrange = wrd.cBurnt + cOrange; // BurntOrange
export const cBurntSienna = wrd.cBurnt + cSienna; // BurntSienna
export const cBurntUmber = wrd.cBurnt + cUmber; // BurntUmber
export const cByzantine = bas.cBy + bas.cza + bas.cnt + phn.cine; // Byzantine
export const cByzantium = wrd.cByzantium; // Byzantium
export const cCadet = wrd.cCadet; // Cadet
export const cCadetBlue = wrd.cCadet + cBlue; // CadetBlue
export const cCadetBlueCrayola = wrd.cCadet + cBlue + wrd.cCrayola; // CadetBlueCrayola
export const cCadetGrey = wrd.cCadet + cGrey; // CadetGrey
export const cCadmiumGreen = cCadmium + cGreen; // CadmiumGreen
export const cCadmiumOrange = cCadmium + cOrange; // CadmiumOrange
export const cCadmiumRed = cCadmium + cRed; // CadmiumRed
export const cCadmiumYellow = cCadmium + cYellow; // CadmiumYellow
export const cCafeAuLait = wrd.cCafe + bas.cAu + bas.cLa + bas.cit; // CafeAuLait
export const cCafeNoir = wrd.cCafe + bas.cNo + bas.cir; // CafeNoir
export const cCambridgeBlue = wrd.cCambridge + cBlue; // CambridgeBlue
export const cCameoPink = wrd.cCameo + cPink; // CameoPink
export const cCanary = wrd.cCan + phn.cary; // Canary
export const cCanaryYellow = wrd.cCan + phn.cary + cYellow; // CanaryYellow
export const cCandyAppleRed = wrd.cCan + bas.cdy + wrd.cApple + cRed; // CandyAppleRed
export const cCandyPink = wrd.cCan + bas.cdy + cPink; // CandyPink
export const cCapri = bas.cCa + phn.cpri; // Capri
export const cCaputMortuum = wrd.cCaput + wrd.cMortuum; // CaputMortuum
export const cCardinal = bas.cCa + bas.crd + phn.cinal; // Cardinal
export const cCaribbeanGreen = wrd.cCaribbean + cGreen; // CaribbeanGreen
export const cCarmine = wrd.cCarmine; // Carmine
export const cCarmineMP = wrd.cCarmine + bas.cMP; // CarmineMP
export const cCarnationPink = wrd.cCarnation + cPink; // CarnationPink
export const cCarnelian = bas.cCa + bas.crn + bas.cel + phn.cian; // Carnelian
export const cCarolinaBlue = bas.cCa + bas.cro + phn.clin + bas.caB + phn.clue; // CarolinaBlue
export const cCarrotOrange = wrd.cCarrot + cOrange; // CarrotOrange
export const cCastletonGreen = wrd.cCastleton + cGreen; // CastletonGreen
export const cCatawba = bas.cCa + bas.cta + bas.cwb + bas.ca; // Catawba
export const cCedarChest = wrd.cCedar + wrd.cChest; // CedarChest
export const cCeladon = wrd.cCeladon; // Celadon
export const cCeladonBlue = wrd.cCeladon + cBlue; // CeladonBlue
export const cCeladonGreen = wrd.cCeladon + cGreen; // CeladonGreen
export const cCeleste = phn.cCel + phn.cest + bas.ce; // Celeste
export const cCelticBlue = wrd.cCeltic + cBlue; // CelticBlue
export const cCerise = wrd.cCerise; // Cerise
export const cCerulean = bas.cCe + phn.crulean; // Cerulean
export const cCeruleanBlue = bas.cCe + phn.crulean + cBlue; // CeruleanBlue
export const cCeruleanFrost = bas.cCe + phn.crulean + wrd.cFrost; // CeruleanFrost
export const cCeruleanCrayola = bas.cCe + phn.crulean + wrd.cCrayola; // CeruleanCrayola
export const cCGBlue = bas.cCG + cBlue; // CGBlue
export const cCGRed = bas.cCG + cRed; // CGRed
export const cChampagne = bas.cCh + phn.campagne; // Champagne
export const cChampagnePink = bas.cCh + phn.campagne + cPink; // ChampagnePink
export const cCharcoal = wrd.cChar + bas.cco + bas.cal; // Charcoal
export const cCharlestonGreen = wrd.cCharleston + cGreen; // CharlestonGreen
export const cCharmPink = wrd.cCharm + cPink; // CharmPink
export const cChartreuseTraditional = wrd.cChartreuse + wrd.cTraditional; // ChartreuseTraditional
export const cChartreuseWeb = wrd.cChartreuse + wrd.cWeb; // ChartreuseWeb
export const cCherryBlossomPink = cCherry + wrd.cBlossom + cPink; // CherryBlossomPink
export const cChestnut = wrd.cChestnut; // Chestnut
export const cChinaPink = ctr.cChina + cPink; // ChinaPink
export const cChinaRose = ctr.cChina + cRose; // ChinaRose
export const cChineseRed = lng.cChinese + cRed; // ChineseRed
export const cChineseViolet = lng.cChinese + cViolet; // ChineseViolet
export const cChineseYellow = lng.cChinese + cYellow; // ChineseYellow
export const cChocolateTraditional = wrd.cChocolate + wrd.cTraditional; // ChocolateTraditional
export const cChocolateWeb = wrd.cChocolate + wrd.cWeb; // ChocolateWeb
export const cChromeYellow = wrd.cChrome + cYellow; // ChromeYellow
export const cCinereous = bas.cCi + bas.cne + bas.cre + phn.cous; // Cinereous
export const cCinnabar = bas.cCi + phn.cnna + bas.cba + bas.cr; // Cinnabar
export const cCinnamonSatin = wrd.cCinnamon + wrd.cSatin; // CinnamonSatin
export const cCitrine = bas.cCi + phn.ctri + bas.cne; // Citrine
export const cCitron = bas.cCi + bas.ctr + bas.con; // Citron
export const cClaret = bas.cCl + phn.care + bas.ct; // Claret
export const cCobaltBlue = cCobalt + cBlue; // CobaltBlue
export const cCocoaBrown = wrd.cCocoa + cBrown; // CocoaBrown
export const cCoffee = wrd.cCoffee; // Coffee
export const cColumbiaBlue = wrd.cColumbia + cBlue; // ColumbiaBlue
export const cCongoPink = ctr.cCongo + cPink; // CongoPink
export const cCoolGrey = wrd.cCool + cGrey; // CoolGrey
export const cCopperCrayola = cCopper + wrd.cCrayola; // CopperCrayola
export const cCopperPenny = cCopper + wrd.cPen + bas.cny; // CopperPenny
export const cCopperRed = cCopper + cRed; // CopperRed
export const cCopperRose = cCopper + cRose; // CopperRose
export const cCoquelicot = bas.cCo + phn.cque + phn.clic + bas.cot; // Coquelicot
export const cCoral = wrd.cCoral; // Coral
export const cCoralPink = wrd.cCoral + cPink; // CoralPink
export const cCordovan = phn.cCor + bas.cdo + wrd.cvan; // Cordovan
export const cCorn = phn.cCor + bas.cn; // Corn
export const cCornflowerBlue = phn.cCor + bas.cnf + wrd.clower + cBlue; // CornflowerBlue
export const cCornsilk = phn.cCor + bas.cns + phn.cilk; // Cornsilk
export const cCosmicCobalt = wrd.cCosmic + cCobalt; // CosmicCobalt
export const cCosmicLatte = wrd.cCosmic + wrd.cLatte; // CosmicLatte
export const cCoyoteBrown = wrd.cCoyote + cBrown; // CoyoteBrown
export const cCottonCandy = wrd.cCotton + wrd.cCan + bas.cdy; // CottonCandy
export const cCream = bas.cCr + phn.ceam; // Cream
export const cCrimsonUA = cCrimson + bas.cUA; // CrimsonUA
export const cCultured = bas.cCu + bas.clt + phn.cured; // Cultured
export const cCyanProcess = cCyan + wrd.cProcess; // CyanProcess
export const cCyberGrape = wrd.cCyber + bas.cGr + phn.cape; // CyberGrape
export const cCyberYellow = wrd.cCyber + cYellow; // CyberYellow
export const cCyclamen = bas.cCy + bas.ccl + phn.camen; // Cyclamen
export const cDarkBlueGray = wrd.cDark + cBlue + cGray; // DarkBlueGray
export const cDarkBrown = wrd.cDark + cBrown; // DarkBrown
export const cDarkByzantium = wrd.cDark + wrd.cByzantium; // DarkByzantium
export const cDarkCornflowerBlue = wrd.cDark + phn.cCor + bas.cnf + wrd.clower + cBlue; // DarkCornflowerBlue
export const cDarkCyan = wrd.cDark + cCyan; // DarkCyan
export const cDarkElectricBlue = wrd.cDark + wrd.cElectric + cBlue; // DarkElectricBlue
export const cDarkGoldenrod = wrd.cDark + cGold + bas.cen + bas.cro + bas.cd; // DarkGoldenrod
export const cDarkGreen = wrd.cDark + cGreen; // DarkGreen
export const cDarkGreenX11 = wrd.cDark + cGreen + bas.cX + num.c11; // DarkGreenX11
export const cDarkJungleGreen = wrd.cDark + bas.cJu + bas.cng + bas.cle + cGreen; // DarkJungleGreen
export const cDarkKhaki = wrd.cDark + wrd.cKhaki; // DarkKhaki
export const cDarkLava = wrd.cDark + bas.cLa + bas.cva; // DarkLava
export const cDarkLiver = wrd.cDark + wrd.cLiver; // DarkLiver
export const cDarkLiverHorses = wrd.cDark + wrd.cLiver + wrd.cHorses; // DarkLiverHorses
export const cDarkMagenta = wrd.cDark + cMagenta; // DarkMagenta
export const cDarkMossGreen = wrd.cDark + wrd.cMoss + cGreen; // DarkMossGreen
export const cDarkOliveGreen = wrd.cDark + cOlive + cGreen; // DarkOliveGreen
export const cDarkOrange = wrd.cDark + cOrange; // DarkOrange
export const cDarkOrchid = wrd.cDark + bas.cOr + phn.cchid; // DarkOrchid
export const cDarkPastelGreen = wrd.cDark + wrd.cPastel + cGreen; // DarkPastelGreen
export const cDarkPurple = wrd.cDark + cPurple; // DarkPurple
export const cDarkRed = wrd.cDark + cRed; // DarkRed
export const cDarkSalmon = wrd.cDark + bas.cSa + phn.clmon; // DarkSalmon
export const cDarkSeaGreen = wrd.cDark + wrd.cSea + cGreen; // DarkSeaGreen
export const cDarkSienna = wrd.cDark + cSienna; // DarkSienna
export const cDarkSkyBlue = wrd.cDark + wrd.cSky + cBlue; // DarkSkyBlue
export const cDarkSlateBlue = wrd.cDark + wrd.cSlate + cBlue; // DarkSlateBlue
export const cDarkSlateGray = wrd.cDark + wrd.cSlate + cGray; // DarkSlateGray
export const cDarkSpringGreen = wrd.cDark + wrd.cSpring + cGreen; // DarkSpringGreen
export const cDarkTurquoise = wrd.cDark + cTurquoise; // DarkTurquoise
export const cDarkViolet = wrd.cDark + cViolet; // DarkViolet
export const cDartmouthGreen = wrd.cDartmouth + cGreen; // DartmouthGreen
export const cDavysGrey = wrd.cDavys + cGrey; // DavysGrey
export const cDeepCerise = wrd.cDeep + wrd.cCerise; // DeepCerise
export const cDeepChampagne = wrd.cDeep + bas.cCh + phn.campagne; // DeepChampagne
export const cDeepChestnut = wrd.cDeep + wrd.cChestnut; // DeepChestnut
export const cDeepJungleGreen = wrd.cDeep + bas.cJu + bas.cng + bas.cle + cGreen; // DeepJungleGreen
export const cDeepPink = wrd.cDeep + cPink; // DeepPink
export const cDeepSaffron = wrd.cDeep + bas.cSa + phn.cffron; // DeepSaffron
export const cDeepSkyBlue = wrd.cDeep + wrd.cSky + cBlue; // DeepSkyBlue
export const cDeepSpaceSparkle = wrd.cDeep + wrd.cSpace + wrd.cSparkle; // DeepSpaceSparkle
export const cDeepTaupe = wrd.cDeep + cTaupe; // DeepTaupe
export const cDenim = bas.cDe + phn.cnim; // Denim
export const cDenimBlue = bas.cDe + phn.cnim + cBlue; // DenimBlue
export const cDesert = bas.cDe + phn.csert; // Desert
export const cDesertSand = bas.cDe + phn.csert + gen.cSan + bas.cd; // DesertSand
export const cDimGray = wrd.cDim + cGray; // DimGray
export const cDodgerBlue = wrd.cDodger + cBlue; // DodgerBlue
export const cDogwoodRose = wrd.cDogwood + cRose; // DogwoodRose
export const cDukeBlue = wrd.cDuke + cBlue; // DukeBlue
export const cDutchWhite = wrd.cDutch + cWhite; // DutchWhite
export const cEarthYellow = wrd.cEarth + cYellow; // EarthYellow
export const cEbony = wrd.cEbony; // Ebony
export const cEcru = bas.cEc + bas.cru; // Ecru
export const cEerieBlack = wrd.cEerie + cBlack; // EerieBlack
export const cEggplant = wrd.cEgg + bas.cpl + phn.cant; // Eggplant
export const cEggshell = wrd.cEgg + wrd.cshell; // Eggshell
export const cEgyptianBlue = ctr.cEgypt + phn.cian + cBlue; // EgyptianBlue
export const cElectricBlue = wrd.cElectric + cBlue; // ElectricBlue
export const cElectricGreen = wrd.cElectric + cGreen; // ElectricGreen
export const cElectricIndigo = wrd.cElectric + cIndigo; // ElectricIndigo
export const cElectricLime = wrd.cElectric + cLime; // ElectricLime
export const cElectricPurple = wrd.cElectric + cPurple; // ElectricPurple
export const cElectricViolet = wrd.cElectric + cViolet; // ElectricViolet
export const cEmerald = bas.cEm + phn.cerald; // Emerald
export const cEminence = bas.cEm + phn.cinen + bas.cce; // Eminence
export const cEnglishGreen = lng.cEnglish + cGreen; // EnglishGreen
export const cEnglishLavender = lng.cEnglish + cLavender; // EnglishLavender
export const cEnglishRed = lng.cEnglish + cRed; // EnglishRed
export const cEnglishVermillion = lng.cEnglish + phn.cVer + num.cmillion; // EnglishVermillion
export const cEnglishViolet = lng.cEnglish + cViolet; // EnglishViolet
export const cErin = bas.cEr + bas.cin; // Erin
export const cEtonBlue = wrd.cEton + cBlue; // EtonBlue
export const cFallow = bas.cFa + phn.cllow; // Fallow
export const cFaluRed = wrd.cFalu + cRed; // FaluRed
export const cFandangoPink = cFandango + cPink; // FandangoPink
export const cFashionFuchsia = wrd.cFashion + cFuchsia; // FashionFuchsia
export const cFawn = bas.cFa + bas.cwn; // Fawn
export const cFeldgrau = bas.cFe + bas.cld + bas.cgr + bas.cau; // Feldgrau
export const cFernGreen = wrd.cFern + cGreen; // FernGreen
export const cFieldDrab = wrd.cField + cDrab; // FieldDrab
export const cFieryRose = wrd.cFiery + cRose; // FieryRose
export const cFirebrick = wrd.cFire + wrd.cbrick; // Firebrick
export const cFireEngineRed = wrd.cFire + wrd.cEngine + cRed; // FireEngineRed
export const cFireOpal = wrd.cFire + bas.cOp + bas.cal; // FireOpal
export const cFlame = bas.cFl + phn.came; // Flame
export const cFlax = bas.cFl + bas.cax; // Flax
export const cFlirt = bas.cFl + bas.cir + bas.ct; // Flirt
export const cFloralWhite = wrd.cFloral + cWhite; // FloralWhite
export const cFluorescentBlue = wrd.cFluorescent + cBlue; // FluorescentBlue
export const cForestGreenCrayola = wrd.cForest + cGreen + wrd.cCrayola; // ForestGreenCrayola
export const cForestGreenTraditional = wrd.cForest + cGreen + wrd.cTraditional; // ForestGreenTraditional
export const cForestGreenWeb = wrd.cForest + cGreen + wrd.cWeb; // ForestGreenWeb
export const cFrenchBeige = lng.cFrench + cBeige; // FrenchBeige
export const cFrenchBistre = lng.cFrench + wrd.cBistre; // FrenchBistre
export const cFrenchBlue = lng.cFrench + cBlue; // FrenchBlue
export const cFrenchFuchsia = lng.cFrench + cFuchsia; // FrenchFuchsia
export const cFrenchLilac = lng.cFrench + cLilac; // FrenchLilac
export const cFrenchLime = lng.cFrench + cLime; // FrenchLime
export const cFrenchMauve = lng.cFrench + cMauve; // FrenchMauve
export const cFrenchPink = lng.cFrench + cPink; // FrenchPink
export const cFrenchRaspberry = lng.cFrench + bas.cRa + bas.csp + wrd.cberry; // FrenchRaspberry
export const cFrenchRose = lng.cFrench + cRose; // FrenchRose
export const cFrenchSkyBlue = lng.cFrench + wrd.cSky + cBlue; // FrenchSkyBlue
export const cFrenchViolet = lng.cFrench + cViolet; // FrenchViolet
export const cFrostbite = wrd.cFrost + wrd.cbite; // Frostbite
export const cFuchsiaCrayola = cFuchsia + wrd.cCrayola; // FuchsiaCrayola
export const cFuchsiaPurple = cFuchsia + cPurple; // FuchsiaPurple
export const cFuchsiaRose = cFuchsia + cRose; // FuchsiaRose
export const cFulvous = phn.cFul + bas.cvo + bas.cus; // Fulvous
export const cFuzzyWuzzy = wrd.cFuzzy + wrd.cWuzzy; // FuzzyWuzzy
export const cGainsboro = wrd.cGain + bas.csb + bas.cor + bas.co; // Gainsboro
export const cGamboge = bas.cGa + bas.cmb + bas.cog + bas.ce; // Gamboge
export const cGenericViridian = wrd.cGeneric + cViridian; // GenericViridian
export const cGhostWhite = wrd.cGhost + cWhite; // GhostWhite
export const cGlaucous = bas.cGl + bas.cau + bas.cco + bas.cus; // Glaucous
export const cGlossyGrape = wrd.cGlossy + bas.cGr + phn.cape; // GlossyGrape
export const cGOGreen = bas.cGO + cGreen; // GOGreen
export const cGoldMetallic = cGold + wrd.cMetallic; // GoldMetallic
export const cGoldWebGolden = cGold + wrd.cWeb + cGold + bas.cen; // GoldWebGolden
export const cGoldCrayola = cGold + wrd.cCrayola; // GoldCrayola
export const cGoldFusion = cGold + wrd.cFusion; // GoldFusion
export const cGoldenBrown = cGold + bas.cen + cBrown; // GoldenBrown
export const cGoldenPoppy = cGold + bas.cen + wrd.cPoppy; // GoldenPoppy
export const cGoldenYellow = cGold + bas.cen + cYellow; // GoldenYellow
export const cGoldenrod = cGold + bas.cen + bas.cro + bas.cd; // Goldenrod
export const cGraniteGray = wrd.cGranite + cGray; // GraniteGray
export const cGrannySmithApple = wrd.cGranny + wrd.cSmith + wrd.cApple; // GrannySmithApple
export const cGrayWeb = cGray + wrd.cWeb; // GrayWeb
export const cGrayX11 = cGray + bas.cX + num.c11; // GrayX11
export const cGreenCrayola = cGreen + wrd.cCrayola; // GreenCrayola
export const cGreenWeb = cGreen + wrd.cWeb; // GreenWeb
export const cGreenMunsell = cGreen + wrd.cMunsell; // GreenMunsell
export const cGreenNCS = cGreen + phn.cNCS; // GreenNCS
export const cGreenPantone = cGreen + wrd.cPantone; // GreenPantone
export const cGreenPigment = cGreen + wrd.cPigment; // GreenPigment
export const cGreenRYB = cGreen + phn.cRYB; // GreenRYB
export const cGreenBlue = cGreen + cBlue; // GreenBlue
export const cGreenBlueCrayola = cGreen + cBlue + wrd.cCrayola; // GreenBlueCrayola
export const cGreenCyan = cGreen + cCyan; // GreenCyan
export const cGreenLizard = cGreen + wrd.cLizard; // GreenLizard
export const cGreenSheen = cGreen + wrd.cSheen; // GreenSheen
export const cGreenYellow = cGreen + cYellow; // GreenYellow
export const cGreenYellowCrayola = cGreen + cYellow + wrd.cCrayola; // GreenYellowCrayola
export const cGrullo = bas.cGr + phn.cull + bas.co; // Grullo
export const cGunmetal = wrd.cGun + wrd.cmeta + bas.cl; // Gunmetal
export const cHanBlue = bas.cHa + bas.cnB + phn.clue; // HanBlue
export const cHanPurple = bas.cHa + bas.cnP + phn.curple; // HanPurple
export const cHansaYellow = bas.cHa + bas.cns + bas.caY + phn.cellow; // HansaYellow
export const cHarlequin = bas.cHa + bas.crl + phn.cequ + bas.cin; // Harlequin
export const cHarvestGold = wrd.cHarvest + cGold; // HarvestGold
export const cHeatWave = wrd.cHeat + wrd.cWave; // HeatWave
export const cHeliotrope = bas.cHe + bas.cli + bas.cot + bas.cro + bas.cpe; // Heliotrope
export const cHeliotropeGray = bas.cHe + bas.cli + bas.cot + bas.cro + bas.cpe + cGray; // HeliotropeGray
export const cHollywoodCerise = wrd.cHollywood + wrd.cCerise; // HollywoodCerise
export const cHoneydew = wrd.cHoney + wrd.cdew; // Honeydew
export const cHonoluluBlue = wrd.cHonolulu + cBlue; // HonoluluBlue
export const cHookersGreen = wrd.cHookers + cGreen; // HookersGreen
export const cHotMagenta = wrd.cHot + cMagenta; // HotMagenta
export const cHotPink = wrd.cHot + cPink; // HotPink
export const cHunterGreen = wrd.cHunter + cGreen; // HunterGreen
export const cIceberg = wrd.cIce + phn.cber + bas.cg; // Iceberg
export const cIcterine = bas.cIc + phn.cter + phn.cine; // Icterine
export const cIlluminatingEmerald = wrd.cIlluminating + bas.cEm + phn.cerald; // IlluminatingEmerald
export const cImperialRed = wrd.cImperial + cRed; // ImperialRed
export const cInchworm = bas.cIn + bas.cch + wrd.cworm; // Inchworm
export const cIndependence = wrd.cIndependence; // Independence
export const cIndiaGreen = gen.cIndia + cGreen; // IndiaGreen
export const cIndianRed = wrd.cIndian + cRed; // IndianRed
export const cIndianYellow = wrd.cIndian + cYellow; // IndianYellow
export const cIndigoDye = cIndigo + wrd.cDye; // IndigoDye
export const cInternationalOrangeAerospace = wrd.cInternational + cOrange + wrd.cAerospace; // InternationalOrangeAerospace
export const cInternationalOrangeEngineering = wrd.cInternational + cOrange + wrd.cEngineering; // InternationalOrangeEngineering
export const cInternationalOrangeGoldenGateBridge = wrd.cInternational + cOrange + cGold + bas.cen + wrd.cGate + wrd.cBridge; // InternationalOrangeGoldenGateBridge
export const cIris = bas.cIr + bas.cis; // Iris
export const cIrresistible = bas.cIr + phn.cres + phn.cist + bas.cib + bas.cle; // Irresistible
export const cIsabelline = bas.cIs + bas.cab + phn.cell + phn.cine; // Isabelline
export const cItalianSkyBlue = lng.cItalian + wrd.cSky + cBlue; // ItalianSkyBlue
export const cIvory = bas.cIv + bas.cor + bas.cy; // Ivory
export const cJade = bas.cJa + bas.cde; // Jade
export const cJapaneseCarmine = lng.cJapanese + wrd.cCarmine; // JapaneseCarmine
export const cJapaneseViolet = lng.cJapanese + cViolet; // JapaneseViolet
export const cJasmine = bas.cJa + bas.csm + phn.cine; // Jasmine
export const cJazzberryJam = wrd.cJazz + wrd.cberry + wrd.cJam; // JazzberryJam
export const cJet = bas.cJe + bas.ct; // Jet
export const cJonquil = bas.cJo + bas.cnq + bas.cui + bas.cl; // Jonquil
export const cJuneBud = gen.cJune + wrd.cBud; // JuneBud
export const cJungleGreen = bas.cJu + bas.cng + bas.cle + cGreen; // JungleGreen
export const cKellyGreen = wrd.cKelly + cGreen; // KellyGreen
export const cKeppel = bas.cKe + bas.cpp + bas.cel; // Keppel
export const cKeyLime = wrd.cKey + cLime; // KeyLime
export const cKhakiWeb = wrd.cKhaki + wrd.cWeb; // KhakiWeb
export const cKhakiX11 = wrd.cKhaki + bas.cX + num.c11; // KhakiX11
export const cKobe = bas.cKo + bas.cbe; // Kobe
export const cKobi = bas.cKo + bas.cbi; // Kobi
export const cKobicha = bas.cKo + bas.cbi + phn.ccha; // Kobicha
export const cKombuGreen = wrd.cKombu + cGreen; // KombuGreen
export const cKSUPurple = bas.cKS + bas.cUP + phn.curple; // KSUPurple
export const cLanguidLavender = wrd.cLanguid + cLavender; // LanguidLavender
export const cLapisLazuli = wrd.cLapis + wrd.cLazuli; // LapisLazuli
export const cLaserLemon = wrd.cLaser + cLemon; // LaserLemon
export const cLaurelGreen = wrd.cLaurel + cGreen; // LaurelGreen
export const cLava = bas.cLa + bas.cva; // Lava
export const cLavenderFloral = cLavender + wrd.cFloral; // LavenderFloral
export const cLavenderWeb = cLavender + wrd.cWeb; // LavenderWeb
export const cLavenderBlue = cLavender + cBlue; // LavenderBlue
export const cLavenderBlush = cLavender + wrd.cBlush; // LavenderBlush
export const cLavenderGray = cLavender + cGray; // LavenderGray
export const cLawnGreen = wrd.cLawn + cGreen; // LawnGreen
export const cLemonChiffon = cLemon + gen.cChi + bas.cff + bas.con; // LemonChiffon
export const cLemonCurry = cLemon + wrd.cCurry; // LemonCurry
export const cLemonGlacier = cLemon + wrd.cGlacier; // LemonGlacier
export const cLemonMeringue = cLemon + wrd.cMeringue; // LemonMeringue
export const cLemonYellow = cLemon + cYellow; // LemonYellow
export const cLemonYellowCrayola = cLemon + cYellow + wrd.cCrayola; // LemonYellowCrayola
export const cLiberty = bas.cLi + phn.cber + bas.cty; // Liberty
export const cLightBlue = wrd.cLight + cBlue; // LightBlue
export const cLightCoral = wrd.cLight + wrd.cCoral; // LightCoral
export const cLightCornflowerBlue = wrd.cLight + phn.cCor + bas.cnf + wrd.clower + cBlue; // LightCornflowerBlue
export const cLightCyan = wrd.cLight + cCyan; // LightCyan
export const cLightFrenchBeige = wrd.cLight + lng.cFrench + cBeige; // LightFrenchBeige
export const cLightGoldenrodYellow = wrd.cLight + cGold + bas.cen + bas.cro + bas.cd + cYellow; // LightGoldenrodYellow
export const cLightGray = wrd.cLight + cGray; // LightGray
export const cLightGreen = wrd.cLight + cGreen; // LightGreen
export const cLightOrange = wrd.cLight + cOrange; // LightOrange
export const cLightPeriwinkle = wrd.cLight + phn.cPer + bas.ciw + phn.cinkle; // LightPeriwinkle
export const cLightPink = wrd.cLight + cPink; // LightPink
export const cLightSalmon = wrd.cLight + bas.cSa + phn.clmon; // LightSalmon
export const cLightSeaGreen = wrd.cLight + wrd.cSea + cGreen; // LightSeaGreen
export const cLightSkyBlue = wrd.cLight + wrd.cSky + cBlue; // LightSkyBlue
export const cLightSlateGray = wrd.cLight + wrd.cSlate + cGray; // LightSlateGray
export const cLightSteelBlue = wrd.cLight + wrd.cSteel + cBlue; // LightSteelBlue
export const cLightYellow = wrd.cLight + cYellow; // LightYellow
export const cLilacLuster = cLilac + wrd.cLuster; // LilacLuster
export const cLimeColorWheel = cLime + wrd.cColor + wrd.cWheel; // LimeColorWheel
export const cLimeWeb = cLime + wrd.cWeb; // LimeWeb
export const cLimeGreen = cLime + cGreen; // LimeGreen
export const cLincolnGreen = wrd.cLincoln + cGreen; // LincolnGreen
export const cLinen = wrd.cLine + bas.cn; // Linen
export const cLion = bas.cLi + bas.con; // Lion
export const cLiseranPurple = wrd.cLiseran + cPurple; // LiseranPurple
export const cLittleBoyBlue = wrd.cLittle + wrd.cBoy + cBlue; // LittleBoyBlue
export const cLiver = wrd.cLiver; // Liver
export const cLiverDogs = wrd.cLiver + wrd.cDogs; // LiverDogs
export const cLiverOrgan = wrd.cLiver + wrd.cOrgan; // LiverOrgan
export const cLiverChestnut = wrd.cLiver + wrd.cChestnut; // LiverChestnut
export const cLivid = bas.cLi + phn.cvid; // Livid
export const cMacaroniAndCheese = wrd.cMacaroni + wrd.cAnd + wrd.cCheese; // MacaroniAndCheese
export const cMadderLake = wrd.cMadder + wrd.cLake; // MadderLake
export const cMagentaCrayola = cMagenta + wrd.cCrayola; // MagentaCrayola
export const cMagentaDye = cMagenta + wrd.cDye; // MagentaDye
export const cMagentaPantone = cMagenta + wrd.cPantone; // MagentaPantone
export const cMagentaProcess = cMagenta + wrd.cProcess; // MagentaProcess
export const cMagentaHaze = cMagenta + wrd.cHaze; // MagentaHaze
export const cMagicMint = wrd.cMagic + gen.cMin + bas.ct; // MagicMint
export const cMagnolia = bas.cMa + bas.cgn + bas.col + bas.cia; // Magnolia
export const cMahogany = bas.cMa + bas.cho + phn.cgan + bas.cy; // Mahogany
export const cMaize = bas.cMa + phn.cize; // Maize
export const cMaizeCrayola = bas.cMa + phn.cize + wrd.cCrayola; // MaizeCrayola
export const cMajorelleBlue = wrd.cMajorelle + cBlue; // MajorelleBlue
export const cMalachite = bas.cMa + phn.clac + phn.chite; // Malachite
export const cManatee = wrd.cMan + phn.cate + bas.ce; // Manatee
export const cMandarin = wrd.cMan + bas.cda + phn.crin; // Mandarin
export const cMango = wrd.cMan + bas.cgo; // Mango
export const cMangoTango = wrd.cMan + bas.cgo + gen.cTango; // MangoTango
export const cMantis = wrd.cMan + phn.ctis; // Mantis
export const cMardiGras = bas.cMa + bas.crd + bas.ciG + phn.cras; // MardiGras
export const cMarigold = bas.cMa + bas.cri + cgold; // Marigold
export const cMaroonCrayola = cMaroon + wrd.cCrayola; // MaroonCrayola
export const cMaroonWeb = cMaroon + wrd.cWeb; // MaroonWeb
export const cMaroonX11 = cMaroon + bas.cX + num.c11; // MaroonX11
export const cMauveTaupe = cMauve + cTaupe; // MauveTaupe
export const cMauvelous = cMauve + bas.clo + bas.cus; // Mauvelous
export const cMaximumBlue = wrd.cMaximum + cBlue; // MaximumBlue
export const cMaximumBlueGreen = wrd.cMaximum + cBlue + cGreen; // MaximumBlueGreen
export const cMaximumBluePurple = wrd.cMaximum + cBlue + cPurple; // MaximumBluePurple
export const cMaximumGreen = wrd.cMaximum + cGreen; // MaximumGreen
export const cMaximumGreenYellow = wrd.cMaximum + cGreen + cYellow; // MaximumGreenYellow
export const cMaximumPurple = wrd.cMaximum + cPurple; // MaximumPurple
export const cMaximumRed = wrd.cMaximum + cRed; // MaximumRed
export const cMaximumRedPurple = wrd.cMaximum + cRed + cPurple; // MaximumRedPurple
export const cMaximumYellow = wrd.cMaximum + cYellow; // MaximumYellow
export const cMaximumYellowRed = wrd.cMaximum + cYellow + cRed; // MaximumYellowRed
export const cMayGreen = gen.cMay + cGreen; // MayGreen
export const cMayaBlue = wrd.cMaya + cBlue; // MayaBlue
export const cMediumAquamarine = wrd.cMedium + cAqua + phn.cmar + phn.cine; // MediumAquamarine
export const cMediumBlue = wrd.cMedium + cBlue; // MediumBlue
export const cMediumCandyAppleRed = wrd.cMedium + wrd.cCan + bas.cdy + wrd.cApple + cRed; // MediumCandyAppleRed
export const cMediumCarmine = wrd.cMedium + wrd.cCarmine; // MediumCarmine
export const cMediumChampagne = wrd.cMedium + bas.cCh + phn.campagne; // MediumChampagne
export const cMediumOrchid = wrd.cMedium + bas.cOr + phn.cchid; // MediumOrchid
export const cMediumPurple = wrd.cMedium + cPurple; // MediumPurple
export const cMediumSeaGreen = wrd.cMedium + wrd.cSea + cGreen; // MediumSeaGreen
export const cMediumSlateBlue = wrd.cMedium + wrd.cSlate + cBlue; // MediumSlateBlue
export const cMediumSpringGreen = wrd.cMedium + wrd.cSpring + cGreen; // MediumSpringGreen
export const cMediumTurquoise = wrd.cMedium + cTurquoise; // MediumTurquoise
export const cMediumVioletRed = wrd.cMedium + cViolet + cRed; // MediumVioletRed
export const cMellowApricot = wrd.cMellow + wrd.cApricot; // MellowApricot
export const cMellowYellow = wrd.cMellow + cYellow; // MellowYellow
export const cMelon = bas.cMe + phn.clon; // Melon
export const cMetallicGold = wrd.cMetallic + cGold; // MetallicGold
export const cMetallicSeaweed = wrd.cMetallic + wrd.cSeaweed; // MetallicSeaweed
export const cMetallicSunburst = wrd.cMetallic + wrd.cSunburst; // MetallicSunburst
export const cMexicanPink = wrd.cMexican + cPink; // MexicanPink
export const cMiddleBlue = wrd.cMiddle + cBlue; // MiddleBlue
export const cMiddleBlueGreen = wrd.cMiddle + cBlue + cGreen; // MiddleBlueGreen
export const cMiddleBluePurple = wrd.cMiddle + cBlue + cPurple; // MiddleBluePurple
export const cMiddleGrey = wrd.cMiddle + cGrey; // MiddleGrey
export const cMiddleGreen = wrd.cMiddle + cGreen; // MiddleGreen
export const cMiddleGreenYellow = wrd.cMiddle + cGreen + cYellow; // MiddleGreenYellow
export const cMiddlePurple = wrd.cMiddle + cPurple; // MiddlePurple
export const cMiddleRed = wrd.cMiddle + cRed; // MiddleRed
export const cMiddleRedPurple = wrd.cMiddle + cRed + cPurple; // MiddleRedPurple
export const cMiddleYellow = wrd.cMiddle + cYellow; // MiddleYellow
export const cMiddleYellowRed = wrd.cMiddle + cYellow + cRed; // MiddleYellowRed
export const cMidnight = bas.cMi + bas.cdn + phn.cight; // Midnight
export const cMidnightBlue = bas.cMi + bas.cdn + phn.cight + cBlue; // MidnightBlue
export const cMidnightGreen = bas.cMi + bas.cdn + phn.cight + cGreen; // MidnightGreen
export const cMikadoYellow = wrd.cMikado + cYellow; // MikadoYellow
export const cMimiPink = wrd.cMimi + cPink; // MimiPink
export const cMindaro = gen.cMin + bas.cda + bas.cro; // Mindaro
export const cMing = gen.cMin + bas.cg; // Ming
export const cMinionYellow = wrd.cMinion + cYellow; // MinionYellow
export const cMint = gen.cMin + bas.ct; // Mint
export const cMintCream = gen.cMin + bas.ctC + phn.cream; // MintCream
export const cMintGreen = gen.cMin + bas.ctG + phn.creen; // MintGreen
export const cMistyMoss = wrd.cMisty + wrd.cMoss; // MistyMoss
export const cMistyRose = wrd.cMisty + cRose; // MistyRose
export const cModeBeige = wrd.cMode + cBeige; // ModeBeige
export const cMorningBlue = wrd.cMorning + cBlue; // MorningBlue
export const cMossGreen = wrd.cMoss + cGreen; // MossGreen
export const cMountainMeadow = wrd.cMountain + wrd.cMeadow; // MountainMeadow
export const cMountbattenPink = wrd.cMount + wrd.cbatten + cPink; // MountbattenPink
export const cMSUGreen = bas.cMS + bas.cUG + phn.creen; // MSUGreen
export const cMulberry = wrd.cMulberry; // Mulberry
export const cMulberryCrayola = wrd.cMulberry + wrd.cCrayola; // MulberryCrayola
export const cMustard = wrd.cMust + phn.card; // Mustard
export const cMyrtleGreen = wrd.cMyrtle + cGreen; // MyrtleGreen
export const cMystic = bas.cMy + phn.cstic; // Mystic
export const cMysticMaroon = bas.cMy + phn.cstic + cMaroon; // MysticMaroon
export const cNadeshikoPink = wrd.cNadeshiko + cPink; // NadeshikoPink
export const cNaplesYellow = wrd.cNaples + cYellow; // NaplesYellow
export const cNavajoWhite = wrd.cNavajo + cWhite; // NavajoWhite
export const cNavyBlue = wrd.cNavy + cBlue; // NavyBlue
export const cNavyBlueCrayola = wrd.cNavy + cBlue + wrd.cCrayola; // NavyBlueCrayola
export const cNeonBlue = wrd.cNeon + cBlue; // NeonBlue
export const cNeonCarrot = wrd.cNeon + wrd.cCarrot; // NeonCarrot
export const cNeonGreen = wrd.cNeon + cGreen; // NeonGreen
export const cNeonFuchsia = wrd.cNeon + cFuchsia; // NeonFuchsia
export const cNewYorkPink = wrd.cNew + wrd.cYork + cPink; // NewYorkPink
export const cNickel = bas.cNi + bas.cc + phn.ckel; // Nickel
export const cNonPhotoBlue = phn.cNon + wrd.cPhoto + cBlue; // NonPhotoBlue
export const cNyanza = bas.cNy + bas.can + bas.cza; // Nyanza
export const cOceanBlue = wrd.cOcean + cBlue; // OceanBlue
export const cOceanGreen = wrd.cOcean + cGreen; // OceanGreen
export const cOchre = bas.cOc + bas.chr + bas.ce; // Ochre
export const cOldBurgundy = wrd.cOld + bas.cBu + phn.crgundy; // OldBurgundy
export const cOldGold = wrd.cOld + cGold; // OldGold
export const cOldLace = wrd.cOld + wrd.cLace; // OldLace
export const cOldLavender = wrd.cOld + cLavender; // OldLavender
export const cOldMauve = wrd.cOld + cMauve; // OldMauve
export const cOldRose = wrd.cOld + cRose; // OldRose
export const cOldSilver = wrd.cOld + cSilver; // OldSilver
export const cOliveDrab3 = cOlive + cDrab + num.c3; // OliveDrab3
export const cOliveDrab7 = cOlive + cDrab + num.c7; // OliveDrab7
export const cOliveGreen = cOlive + cGreen; // OliveGreen
export const cOlivine = bas.cOl + bas.civ + phn.cine; // Olivine
export const cOnyx = bas.cOn + bas.cyx; // Onyx
export const cOpal = bas.cOp + bas.cal; // Opal
export const cOperaMauve = wrd.cOpera + cMauve; // OperaMauve
export const cOrangeCrayola = cOrange + wrd.cCrayola; // OrangeCrayola
export const cOrangePantone = cOrange + wrd.cPantone; // OrangePantone
export const cOrangeWeb = cOrange + wrd.cWeb; // OrangeWeb
export const cOrangePeel = cOrange + wrd.cPeel; // OrangePeel
export const cOrangeRed = cOrange + cRed; // OrangeRed
export const cOrangeRedCrayola = cOrange + cRed + wrd.cCrayola; // OrangeRedCrayola
export const cOrangeSoda = cOrange + wrd.cSoda; // OrangeSoda
export const cOrangeYellow = cOrange + cYellow; // OrangeYellow
export const cOrangeYellowCrayola = cOrange + cYellow + wrd.cCrayola; // OrangeYellowCrayola
export const cOrchid = bas.cOr + phn.cchid; // Orchid
export const cOrchidPink = bas.cOr + phn.cchid + cPink; // OrchidPink
export const cOrchidCrayola = bas.cOr + phn.cchid + wrd.cCrayola; // OrchidCrayola
export const cOuterSpaceCrayola = wrd.cOuter + wrd.cSpace + wrd.cCrayola; // OuterSpaceCrayola
export const cOutrageousOrange = wrd.cOutrageous + cOrange; // OutrageousOrange
export const cOxBlood = bas.cOx + wrd.cBlood; // OxBlood
export const cOxfordBlue = wrd.cOxford + cBlue; // OxfordBlue
export const cOUCrimsonRed = bas.cOU + cCrimson + cRed; // OUCrimsonRed
export const cPacificBlue = wrd.cPacific + cBlue; // PacificBlue
export const cPakistanGreen = ctr.cPakistan + cGreen; // PakistanGreen
export const cPalatinatePurple = wrd.cPalatinate + cPurple; // PalatinatePurple
export const cPaleAqua = wrd.cPale + cAqua; // PaleAqua
export const cPaleCerulean = wrd.cPale + bas.cCe + phn.crulean; // PaleCerulean
export const cPalePink = wrd.cPale + cPink; // PalePink
export const cPalePurplePantone = wrd.cPale + cPurple + wrd.cPantone; // PalePurplePantone
export const cPaleSilver = wrd.cPale + cSilver; // PaleSilver
export const cPaleSpringBud = wrd.cPale + wrd.cSpring + wrd.cBud; // PaleSpringBud
export const cPansyPurple = wrd.cPansy + cPurple; // PansyPurple
export const cPaoloVeroneseGreen = wrd.cPaolo + wrd.cVeronese + cGreen; // PaoloVeroneseGreen
export const cPapayaWhip = wrd.cPapaya + wrd.cWhip; // PapayaWhip
export const cParadisePink = wrd.cParadise + cPink; // ParadisePink
export const cParisGreen = wrd.cParis + cGreen; // ParisGreen
export const cPastelPink = wrd.cPastel + cPink; // PastelPink
export const cPatriarch = wrd.cPat + phn.cria + phn.crch; // Patriarch
export const cPaynesGrey = wrd.cPaynes + cGrey; // PaynesGrey
export const cPeachCrayola = cPeach + wrd.cCrayola; // PeachCrayola
export const cPeachPuff = cPeach + wrd.cPuff; // PeachPuff
export const cPear = bas.cPe + bas.car; // Pear
export const cPearlyPurple = wrd.cPearly + cPurple; // PearlyPurple
export const cPeriwinkle = phn.cPer + bas.ciw + phn.cinkle; // Periwinkle
export const cPeriwinkleCrayola = phn.cPer + bas.ciw + phn.cinkle + wrd.cCrayola; // PeriwinkleCrayola
export const cPermanentGeraniumLake = wrd.cPermanent + wrd.cGeranium + wrd.cLake; // PermanentGeraniumLake
export const cPersianBlue = wrd.cPersian + cBlue; // PersianBlue
export const cPersianGreen = wrd.cPersian + cGreen; // PersianGreen
export const cPersianIndigo = wrd.cPersian + cIndigo; // PersianIndigo
export const cPersianOrange = wrd.cPersian + cOrange; // PersianOrange
export const cPersianPink = wrd.cPersian + cPink; // PersianPink
export const cPersianPlum = wrd.cPersian + cPlum; // PersianPlum
export const cPersianRed = wrd.cPersian + cRed; // PersianRed
export const cPersianRose = wrd.cPersian + cRose; // PersianRose
export const cPersimmon = phn.cPer + phn.csim + phn.cmon; // Persimmon
export const cPewterBlue = wrd.cPewter + cBlue; // PewterBlue
export const cPhlox = bas.cPh + bas.clo + bas.cx; // Phlox
export const cPhthaloBlue = wrd.cPhthalo + cBlue; // PhthaloBlue
export const cPhthaloGreen = wrd.cPhthalo + cGreen; // PhthaloGreen
export const cPicoteeBlue = wrd.cPicotee + cBlue; // PicoteeBlue
export const cPictorialCarmine = wrd.cPictorial + wrd.cCarmine; // PictorialCarmine
export const cPiggyPink = wrd.cPiggy + cPink; // PiggyPink
export const cPineGreen = wrd.cPine + cGreen; // PineGreen
export const cPineTree = wrd.cPine + wrd.cTree; // PineTree
export const cPinkPantone = cPink + wrd.cPantone; // PinkPantone
export const cPinkFlamingo = cPink + wrd.cFlamingo; // PinkFlamingo
export const cPinkLace = cPink + wrd.cLace; // PinkLace
export const cPinkLavender = cPink + cLavender; // PinkLavender
export const cPinkSherbet = cPink + wrd.cSherbet; // PinkSherbet
export const cPistachio = bas.cPi + bas.cst + phn.cach + bas.cio; // Pistachio
export const cPlumWeb = cPlum + wrd.cWeb; // PlumWeb
export const cPlumpPurple = wrd.cPlump + cPurple; // PlumpPurple
export const cPolishedPine = wrd.cPolished + wrd.cPine; // PolishedPine
export const cPompAndPower = wrd.cPomp + wrd.cAnd + wrd.cPower; // PompAndPower
export const cPopstar = wrd.cPop + phn.cstar; // Popstar
export const cPortlandOrange = wrd.cPortland + cOrange; // PortlandOrange
export const cPowderBlue = wrd.cPowder + cBlue; // PowderBlue
export const cPrincetonOrange = wrd.cPrinceton + cOrange; // PrincetonOrange
export const cPrune = bas.cPr + phn.cune; // Prune
export const cPrussianBlue = wrd.cPrussian + cBlue; // PrussianBlue
export const cPsychedelicPurple = wrd.cPsychedelic + cPurple; // PsychedelicPurple
export const cPuce = bas.cPu + bas.cce; // Puce
export const cPullmanBrown = wrd.cPullman + cBrown; // PullmanBrown
export const cPumpkin = bas.cPu + bas.cmp + phn.ckin; // Pumpkin
export const cPurpleWeb = cPurple + wrd.cWeb; // PurpleWeb
export const cPurpleMunsell = cPurple + wrd.cMunsell; // PurpleMunsell
export const cPurpleX11 = cPurple + bas.cX + num.c11; // PurpleX11
export const cPurpleMountainMajesty = cPurple + wrd.cMountain + wrd.cMajesty; // PurpleMountainMajesty
export const cPurpleNavy = cPurple + wrd.cNavy; // PurpleNavy
export const cPurplePizzazz = cPurple + wrd.cPizzazz; // PurplePizzazz
export const cPurplePlum = cPurple + cPlum; // PurplePlum
export const cPurpureus = bas.cPu + bas.crp + phn.cure + bas.cus; // Purpureus
export const cQueenBlue = wrd.cQueen + cBlue; // QueenBlue
export const cQueenPink = wrd.cQueen + cPink; // QueenPink
export const cQuickSilver = wrd.cQuick + cSilver; // QuickSilver
export const cQuinacridoneMagenta = wrd.cQuinacridone + cMagenta; // QuinacridoneMagenta
export const cRadicalRed = wrd.cRadical + cRed; // RadicalRed
export const cRaisinBlack = wrd.cRaisin + cBlack; // RaisinBlack
export const cRajah = bas.cRa + bas.cja + bas.ch; // Rajah
export const cRaspberry = bas.cRa + bas.csp + wrd.cberry; // Raspberry
export const cRaspberryGlace = bas.cRa + bas.csp + wrd.cberry + wrd.cGlace; // RaspberryGlace
export const cRaspberryRose = bas.cRa + bas.csp + wrd.cberry + cRose; // RaspberryRose
export const cRawSienna = wrd.cRaw + cSienna; // RawSienna
export const cRawUmber = wrd.cRaw + cUmber; // RawUmber
export const cRazzleDazzleRose = wrd.cRazzle + wrd.cDazzle + cRose; // RazzleDazzleRose
export const cRazzmatazz = bas.cRa + bas.czz + wrd.cmat + phn.cazz; // Razzmatazz
export const cRazzmicBerry = wrd.cRazzmic + wrd.cBerry; // RazzmicBerry
export const cRebeccaPurple = wrd.cRebecca + cPurple; // RebeccaPurple
export const cRedCrayola = cRed + wrd.cCrayola; // RedCrayola
export const cRedMunsell = cRed + wrd.cMunsell; // RedMunsell
export const cRedNCS = cRed + phn.cNCS; // RedNCS
export const cRedPantone = cRed + wrd.cPantone; // RedPantone
export const cRedPigment = cRed + wrd.cPigment; // RedPigment
export const cRedRYB = cRed + phn.cRYB; // RedRYB
export const cRedOrange = cRed + cOrange; // RedOrange
export const cRedOrangeCrayola = cRed + cOrange + wrd.cCrayola; // RedOrangeCrayola
export const cRedOrangeColorWheel = cRed + cOrange + wrd.cColor + wrd.cWheel; // RedOrangeColorWheel
export const cRedPurple = cRed + cPurple; // RedPurple
export const cRedSalsa = cRed + wrd.cSalsa; // RedSalsa
export const cRedViolet = cRed + cViolet; // RedViolet
export const cRedVioletCrayola = cRed + cViolet + wrd.cCrayola; // RedVioletCrayola
export const cRedVioletColorWheel = cRed + cViolet + wrd.cColor + wrd.cWheel; // RedVioletColorWheel
export const cRedwood = cRed + wrd.cwood; // Redwood
export const cResolutionBlue = wrd.cResolution + cBlue; // ResolutionBlue
export const cRhythm = bas.cRh + bas.cyt + bas.chm; // Rhythm
export const cRichBlack = wrd.cRich + cBlack; // RichBlack
export const cRichBlackFOGRA29 = wrd.cRich + cBlack + bas.cFO + bas.cGR + bas.cA + num.c29; // RichBlackFOGRA29
export const cRichBlackFOGRA39 = wrd.cRich + cBlack + bas.cFO + bas.cGR + bas.cA + num.c39; // RichBlackFOGRA39
export const cRifleGreen = wrd.cRifle + cGreen; // RifleGreen
export const cRobinEggBlue = wrd.cRobin + wrd.cEgg + cBlue; // RobinEggBlue
export const cRocketMetallic = wrd.cRocket + wrd.cMetallic; // RocketMetallic
export const cRomanSilver = wrd.cRoman + cSilver; // RomanSilver
export const cRoseBonbon = cRose + wrd.cBonbon; // RoseBonbon
export const cRoseDust = cRose + wrd.cDust; // RoseDust
export const cRoseEbony = cRose + wrd.cEbony; // RoseEbony
export const cRoseMadder = cRose + wrd.cMadder; // RoseMadder
export const cRosePink = cRose + cPink; // RosePink
export const cRoseQuartz = cRose + wrd.cQuartz; // RoseQuartz
export const cRoseRed = cRose + cRed; // RoseRed
export const cRoseTaupe = cRose + cTaupe; // RoseTaupe
export const cRoseVale = cRose + wrd.cVale; // RoseVale
export const cRosewood = cRose + wrd.cwood; // Rosewood
export const cRossoCorsa = wrd.cRosso + wrd.cCorsa; // RossoCorsa
export const cRosyBrown = wrd.cRosy + cBrown; // RosyBrown
export const cRoyalBlueDark = wrd.cRoyal + cBlue + wrd.cDark; // RoyalBlueDark
export const cRoyalBlueLight = wrd.cRoyal + cBlue + wrd.cLight; // RoyalBlueLight
export const cRoyalPurple = wrd.cRoyal + cPurple; // RoyalPurple
export const cRoyalYellow = wrd.cRoyal + cYellow; // RoyalYellow
export const cRuber = wrd.cRub + bas.cer; // Ruber
export const cRubineRed = wrd.cRubine + cRed; // RubineRed
export const cRubyRed = cRuby + cRed; // RubyRed
export const cRufous = bas.cRu + bas.cfo + bas.cus; // Rufous
export const cRusset = bas.cRu + bas.css + bas.cet; // Russet
export const cRussianGreen = lng.cRussian + cGreen; // RussianGreen
export const cRussianViolet = lng.cRussian + cViolet; // RussianViolet
export const cRust = bas.cRu + bas.cst; // Rust
export const cRustyRed = bas.cRu + phn.csty + cRed; // RustyRed
export const cSacramentoStateGreen = wrd.cSacramento + wrd.cState + cGreen; // SacramentoStateGreen
export const cSaddleBrown = wrd.cSaddle + cBrown; // SaddleBrown
export const cSafetyOrange = wrd.cSafety + cOrange; // SafetyOrange
export const cBlazeOrange = wrd.cBlaze + cOrange; // BlazeOrange
export const cSafetyYellow = wrd.cSafety + cYellow; // SafetyYellow
export const cSaffron = bas.cSa + phn.cffron; // Saffron
export const cSage = bas.cSa + bas.cge; // Sage
export const cStPatricksBlue = bas.cSt + wrd.cPatricks + cBlue; // StPatricksBlue
export const cSalmon = bas.cSa + phn.clmon; // Salmon
export const cSalmonPink = bas.cSa + phn.clmon + cPink; // SalmonPink
export const cSand = gen.cSan + bas.cd; // Sand
export const cSandDune = gen.cSan + bas.cdD + phn.cune; // SandDune
export const cSandyBrown = wrd.cSandy + cBrown; // SandyBrown
export const cSapGreen = wrd.cSap + cGreen; // SapGreen
export const cSapphireBlue = cSapphire + cBlue; // SapphireBlue
export const cSapphireCrayola = cSapphire + wrd.cCrayola; // SapphireCrayola
export const cSatinSheenGold = wrd.cSatin + wrd.cSheen + cGold; // SatinSheenGold
export const cSchaussPink = wrd.cSchauss + cPink; // SchaussPink
export const cSchoolBusYellow = wrd.cSchool + wrd.cBus + cYellow; // SchoolBusYellow
export const cScreaminGreen = wrd.cScreamin + cGreen; // ScreaminGreen
export const cSeaGreen = wrd.cSea + cGreen; // SeaGreen
export const cSeaGreenCrayola = wrd.cSea + cGreen + wrd.cCrayola; // SeaGreenCrayola
export const cSealBrown = wrd.cSeal + cBrown; // SealBrown
export const cSeashell = wrd.cSea + wrd.cshell; // Seashell
export const cSelectiveYellow = wrd.cSelective + cYellow; // SelectiveYellow
export const cSepia = bas.cSe + phn.cpia; // Sepia
export const cShadow = phn.cSha + phn.cdow; // Shadow
export const cShadowBlue = phn.cSha + phn.cdow + cBlue; // ShadowBlue
export const cShamrockGreen = wrd.cShamrock + cGreen; // ShamrockGreen
export const cSheenGreen = wrd.cSheen + cGreen; // SheenGreen
export const cShimmeringBlush = wrd.cShimmering + wrd.cBlush; // ShimmeringBlush
export const cShinyShamrock = wrd.cShiny + wrd.cShamrock; // ShinyShamrock
export const cShockingPink = wrd.cShocking + cPink; // ShockingPink
export const cShockingPinkCrayola = wrd.cShocking + cPink + wrd.cCrayola; // ShockingPinkCrayola
export const cSilverCrayola = cSilver + wrd.cCrayola; // SilverCrayola
export const cSilverMetallic = cSilver + wrd.cMetallic; // SilverMetallic
export const cSilverChalice = cSilver + wrd.cChalice; // SilverChalice
export const cSilverPink = cSilver + cPink; // SilverPink
export const cSilverSand = cSilver + gen.cSan + bas.cd; // SilverSand
export const cSinopia = bas.cSi + bas.cno + phn.cpia; // Sinopia
export const cSizzlingRed = wrd.cSizzling + cRed; // SizzlingRed
export const cSizzlingSunrise = wrd.cSizzling + wrd.cSunrise; // SizzlingSunrise
export const cSkobeloff = bas.cSk + phn.cobe + bas.clo + bas.cff; // Skobeloff
export const cSkyBlue = wrd.cSky + cBlue; // SkyBlue
export const cSkyBlueCrayola = wrd.cSky + cBlue + wrd.cCrayola; // SkyBlueCrayola
export const cSkyMagenta = wrd.cSky + cMagenta; // SkyMagenta
export const cSlateBlue = wrd.cSlate + cBlue; // SlateBlue
export const cSlateGray = wrd.cSlate + cGray; // SlateGray
export const cSlimyGreen = wrd.cSlimy + cGreen; // SlimyGreen
export const cSmitten = bas.cSm + bas.cit + num.cten; // Smitten
export const cSmokyBlack = wrd.cSmoky + cBlack; // SmokyBlack
export const cSnow = bas.cSn + bas.cow; // Snow
export const cSolidPink = wrd.cSolid + cPink; // SolidPink
export const cSonicSilver = wrd.cSonic + cSilver; // SonicSilver
export const cSpaceCadet = wrd.cSpace + wrd.cCadet; // SpaceCadet
export const cSpanishBistre = lng.cSpanish + wrd.cBistre; // SpanishBistre
export const cSpanishBlue = lng.cSpanish + cBlue; // SpanishBlue
export const cSpanishCarmine = lng.cSpanish + wrd.cCarmine; // SpanishCarmine
export const cSpanishGray = lng.cSpanish + cGray; // SpanishGray
export const cSpanishGreen = lng.cSpanish + cGreen; // SpanishGreen
export const cSpanishOrange = lng.cSpanish + cOrange; // SpanishOrange
export const cSpanishPink = lng.cSpanish + cPink; // SpanishPink
export const cSpanishRed = lng.cSpanish + cRed; // SpanishRed
export const cSpanishSkyBlue = lng.cSpanish + wrd.cSky + cBlue; // SpanishSkyBlue
export const cSpanishViolet = lng.cSpanish + cViolet; // SpanishViolet
export const cSpanishViridian = lng.cSpanish + cViridian; // SpanishViridian
export const cSpringBud = wrd.cSpring + wrd.cBud; // SpringBud
export const cSpringFrost = wrd.cSpring + wrd.cFrost; // SpringFrost
export const cSpringGreen = wrd.cSpring + cGreen; // SpringGreen
export const cSpringGreenCrayola = wrd.cSpring + cGreen + wrd.cCrayola; // SpringGreenCrayola
export const cStarCommandBlue = wrd.cStar + wrd.cCommand + cBlue; // StarCommandBlue
export const cSteelBlue = wrd.cSteel + cBlue; // SteelBlue
export const cSteelPink = wrd.cSteel + cPink; // SteelPink
export const cSteelTeal = wrd.cSteel + cTeal; // SteelTeal
export const cStilDeGrainYellow = wrd.cStil + bas.cDe + wrd.cGrain + cYellow; // StilDeGrainYellow
export const cStraw = bas.cSt + phn.craw; // Straw
export const cSugarPlum = wrd.cSugar + cPlum; // SugarPlum
export const cSunglow = wrd.cSun + wrd.cglow; // Sunglow
export const cSunray = wrd.cSun + phn.cray; // Sunray
export const cSunset = wrd.cSuns + bas.cet; // Sunset
export const cSuperPink = wrd.cSuper + cPink; // SuperPink
export const cSweetBrown = wrd.cSweet + cBrown; // SweetBrown
export const cTanCrayola = cTan + wrd.cCrayola; // TanCrayola
export const cTangoPink = gen.cTango + cPink; // TangoPink
export const cTartOrange = phn.cTart + cOrange; // TartOrange
export const cTaupeGray = cTaupe + cGray; // TaupeGray
export const cTeaGreen = wrd.cTea + cGreen; // TeaGreen
export const cTeaRose = wrd.cTea + cRose; // TeaRose
export const cTeaRoseWeb = wrd.cTea + cRose + wrd.cWeb; // TeaRoseWeb
export const cTealBlue = cTeal + cBlue; // TealBlue
export const cTelemagenta = bas.cTe + bas.cle + cmagenta; // Telemagenta
export const cTawny = bas.cTa + bas.cwn + bas.cy; // Tawny
export const cTerraCotta = wrd.cTerra + wrd.cCotta; // TerraCotta
export const cThistle = wrd.cThis + phn.ctle; // Thistle
export const cThulianPink = wrd.cThulian + cPink; // ThulianPink
export const cTickleMePink = wrd.cTickle + bas.cMe + cPink; // TickleMePink
export const cTiffanyBlue = wrd.cTiffany + cBlue; // TiffanyBlue
export const cTimberwolf = wrd.cTimber + wrd.cwolf; // Timberwolf
export const cTitaniumYellow = cTitanium + cYellow; // TitaniumYellow
export const cTomato = bas.cTo + wrd.cmat + bas.co; // Tomato
export const cTropicalRainforest = wrd.cTropical + wrd.cRain + phn.cfore + bas.cst; // TropicalRainforest
export const cTrueBlue = gen.cTrue + cBlue; // TrueBlue
export const cTrypanBlue = wrd.cTrypan + cBlue; // TrypanBlue
export const cTuftsBlue = wrd.cTufts + cBlue; // TuftsBlue
export const cTumbleweed = wrd.cTumble + wrd.cweed; // Tumbleweed
export const cTurquoiseBlue = cTurquoise + cBlue; // TurquoiseBlue
export const cTurquoiseGreen = cTurquoise + cGreen; // TurquoiseGreen
export const cTurtleGreen = bas.cTu + bas.crt + bas.cle + cGreen; // TurtleGreen
export const cTuscanBrown = cTuscan + cBrown; // TuscanBrown
export const cTuscanRed = cTuscan + cRed; // TuscanRed
export const cTuscanTan = cTuscan + cTan; // TuscanTan
export const cTuscany = cTuscan + bas.cy; // Tuscany
export const cTwilightLavender = wrd.cTwilight + cLavender; // TwilightLavender
export const cTyrianPurple = wrd.cTyrian + cPurple; // TyrianPurple
export const cUABlue = bas.cUA + cBlue; // UABlue
export const cUARed = bas.cUA + cRed; // UARed
export const cUltramarine = wrd.cUltra + phn.cmar + phn.cine; // Ultramarine
export const cUltramarineBlue = wrd.cUltra + phn.cmar + phn.cine + cBlue; // UltramarineBlue
export const cUltraPink = wrd.cUltra + cPink; // UltraPink
export const cUltraRed = wrd.cUltra + cRed; // UltraRed
export const cUnbleachedSilk = wrd.cUnbleached + wrd.cSilk; // UnbleachedSilk
export const cUnitedNationsBlue = wrd.cUnited + wrd.cNations + cBlue; // UnitedNationsBlue
export const cUnmellowYellow = bas.cUn + phn.cmel + wrd.clow + cYellow; // UnmellowYellow
export const cUPForestGreen = bas.cUP + wrd.cForest + cGreen; // UPForestGreen
export const cUPMaroon = bas.cUP + cMaroon; // UPMaroon
export const cUpsdellRed = wrd.cUpsdell + cRed; // UpsdellRed
export const cUranianBlue = wrd.cUranian + cBlue; // UranianBlue
export const cUSAFABlue = bas.cUS + bas.cAF + bas.cAB + phn.clue; // USAFABlue
export const cVanDykeBrown = wrd.cVan + wrd.cDyke + cBrown; // VanDykeBrown
export const cVanillaIce = cVanilla + wrd.cIce; // VanillaIce
export const cVegasGold = wrd.cVegas + cGold; // VegasGold
export const cVenetianRed = wrd.cVenetian + cRed; // VenetianRed
export const cVerdigris = phn.cVer + wrd.cdig + phn.cris; // Verdigris
export const cVermilionWeb = cVermilion + wrd.cWeb; // VermilionWeb
export const cVeronica = phn.cVer + bas.con + phn.cica; // Veronica
export const cVioletColorWheel = cViolet + wrd.cColor + wrd.cWheel; // VioletColorWheel
export const cVioletCrayola = cViolet + wrd.cCrayola; // VioletCrayola
export const cVioletRYB = cViolet + phn.cRYB; // VioletRYB
export const cVioletWeb = cViolet + wrd.cWeb; // VioletWeb
export const cVioletBlue = cViolet + cBlue; // VioletBlue
export const cVioletBlueCrayola = cViolet + cBlue + wrd.cCrayola; // VioletBlueCrayola
export const cVioletRed = cViolet + cRed; // VioletRed
export const cViridianGreen = cViridian + cGreen; // ViridianGreen
export const cVividBurgundy = wrd.cVivid + bas.cBu + phn.crgundy; // VividBurgundy
export const cVividSkyBlue = wrd.cVivid + wrd.cSky + cBlue; // VividSkyBlue
export const cVividTangerine = wrd.cVivid + cTangerine; // VividTangerine
export const cVividViolet = wrd.cVivid + cViolet; // VividViolet
export const cVolt = bas.cVo + bas.clt; // Volt
export const cWarmBlack = wrd.cWarm + cBlack; // WarmBlack
export const cWheat = bas.cWh + phn.ceat; // Wheat
export const cWildBlueYonder = wrd.cWild + cBlue + wrd.cYonder; // WildBlueYonder
export const cWildOrchid = wrd.cWild + bas.cOr + phn.cchid; // WildOrchid
export const cWildStrawberry = wrd.cWild + bas.cSt + phn.craw + wrd.cberry; // WildStrawberry
export const cWildWatermelon = wrd.cWild + bas.cWa + wrd.cterm + phn.celon; // WildWatermelon
export const cWindsorTan = wrd.cWindsor + cTan; // WindsorTan
export const cWine = bas.cWi + bas.cne; // Wine
export const cWineDregs = bas.cWi + bas.cne + wrd.cDregs; // WineDregs
export const cWinterSky = wrd.cWinter + wrd.cSky; // WinterSky
export const cWintergreenDream = wrd.cWinter + cgreen + wrd.cDream; // WintergreenDream
export const cWisteria = bas.cWi + phn.cster + bas.cia; // Wisteria
export const cWoodBrown = bas.cWo + bas.cod + cBrown; // WoodBrown
export const cXanthic = bas.cXa + wrd.cnth + bas.cic; // Xanthic
export const cXanadu = bas.cXa + bas.cna + bas.cdu; // Xanadu
export const cYaleBlue = wrd.cYale + cBlue; // YaleBlue
export const cYellowCrayola = cYellow + wrd.cCrayola; // YellowCrayola
export const cYellowMunsell = cYellow + wrd.cMunsell; // YellowMunsell
export const cYellowNCS = cYellow + phn.cNCS; // YellowNCS
export const cYellowPantone = cYellow + wrd.cPantone; // YellowPantone
export const cYellowProcess = cYellow + wrd.cProcess; // YellowProcess
export const cYellowRYB = cYellow + phn.cRYB; // YellowRYB
export const cYellowGreen = cYellow + cGreen; // YellowGreen
export const cYellowGreenCrayola = cYellow + cGreen + wrd.cCrayola; // YellowGreenCrayola
export const cYellowGreenColorWheel = cYellow + cGreen + wrd.cColor + wrd.cWheel; // YellowGreenColorWheel
export const cYellowOrange = cYellow + cOrange; // YellowOrange
export const cYellowOrangeColorWheel = cYellow + cOrange + wrd.cColor + wrd.cWheel; // YellowOrangeColorWheel
export const cYellowSunshine = cYellow + wrd.cSuns + phn.chine; // YellowSunshine
export const cYInMnBlue = bas.cY + bas.cIn + bas.cMn + cBlue; // YInMnBlue
export const cZaffre = bas.cZa + bas.cff + bas.cre; // Zaffre
export const cZomp = bas.cZo + bas.cmp; // Zomp
