// @haystacks/constants/src/constants/element.constants.js
/**
 * @file element.constants
 * @module element.constants
 * @description Contains named elements of the periodic table of elements.
 * @requires module:basic.constants
 * @requires module:generic.constants
 * @requires module:numeric.constants
 * @requires module:phonic.constants
 * @requires module:word.constants
 * @author Seth Hollingsead
 * @date 2022/04/14
 * @copyright Copyright © 2022-… by Seth Hollingsead. All rights reserved
 */

// Internal imports
import * as bas from './basic.constants.js';
import * as gen from './generic.constants.js';
import * as num from './numeric.constants.js';
import * as phn from './phonic.constants.js';
import * as wrd from './word.constants.js';

export const cH = bas.cH; // H
export const cHe = bas.cHe; // He
export const cLi = bas.cLi; // Li
export const cBe = bas.cBe; // Be
export const cB = bas.cB; // B
export const cC = bas.cC; // C
export const cN = bas.cN; // N
export const cO = bas.cO; // O
export const cF = bas.cF; // F
export const cNe = bas.cNe; // Ne
export const cNa = bas.cNa; // Na
export const cMg = bas.cMg; // Mg
export const cAl = bas.cAl; // Al
export const cSi = bas.cSi; // Si
export const cP = bas.cP; // P
export const cS = bas.cS; // S
export const cCl = bas.cCl; // Cl
export const cAr = bas.cAr; // Ar
export const cK = bas.cK; // K
export const cCa = bas.cCa; // Ca
export const cSc = bas.cSc; // Sc
export const cTi = bas.cTi; // Ti
export const cV = bas.cV; // V
export const cCr = bas.cCr; // Cr
export const cMn = bas.cMn; // Mn
export const cFe = bas.cFe; // Fe
export const cCo = bas.cCo; // Co
export const cNi = bas.cNi; // Ni
export const cCu = bas.cCu; // Cu
export const cZn = bas.cZn; // Zn
export const cGa = bas.cGa; // Ga
export const cGe = bas.cGe; // Ge
export const cAs = bas.cAs; // As
export const cSe = bas.cSe; // Se
export const cBr = bas.cBr; // Br
export const cKr = bas.cKr; // Kr
export const cRb = bas.cRb; // Rb
export const cSr = bas.cSr; // Sr
export const cY = bas.cY; // Y
export const cZr = bas.cZr; // Zr
export const cNb = bas.cNb; // Nb
export const cMo = bas.cMo; // Mo
export const cTc = bas.cTc; // Tc
export const cRu = bas.cRu; // Ru
export const cRh = bas.cRh; // Rh
export const cPd = bas.cPd; // Pd
export const cAg = bas.cAg; // Ag
export const cCd = bas.cCd; // Cd
export const cIn = bas.cIn; // In
export const cSn = bas.cSn; // Sn
export const cSb = bas.cSb; // Sb
export const cTe = bas.cTe; // Te
export const cI = bas.cI; // I
export const cXe = bas.cXe; // Xe
export const cCs = bas.cCs; // Cs
export const cBa = bas.cBa; // Ba
export const cLa = bas.cLa; // La
export const cCe = bas.cCe; // Ce
export const cPr = bas.cPr; // Pr
export const cNd = bas.cNd; // Nd
export const cPm = bas.cPm; // Pm
export const cSm = bas.cSm; // Sm
export const cEu = bas.cEu; // Eu
export const cGd = bas.cGd; // Gd
export const cTb = bas.cTb; // Tb
export const cDy = bas.cDy; // Dy
export const cHo = bas.cHo; // Ho
export const cEr = bas.cEr; // Er
export const cTm = bas.cTm; // Tm
export const cYb = bas.cYb; // Yb
export const cLu = bas.cLu; // Lu
export const cHf = bas.cHf; // Hf
export const cTa = bas.cTa; // Ta
export const cW = bas.cW; // W
export const cRe = bas.cRe; // Re
export const cOs = bas.cOs; // Os
export const cIr = bas.cIr; // Ir
export const cPt = bas.cPt; // Pt
export const cAu = bas.cAu; // Au
export const cHg = bas.cHg; // Hg
export const cTl = bas.cTl; // Tl
export const cPb = bas.cPb; // Pb
export const cBi = bas.cBi; // Bi
export const cPo = bas.cPo; // Po
export const cAt = bas.cA + bas.ct; // At
export const cRn = bas.cRn; // Rn
export const cFr = bas.cFr; // Fr
export const cRa = bas.cRa; // Ra
export const cAc = bas.cAc; // Ac
export const cTh = bas.cTh; // Th
export const cPa = bas.cPa; // Pa
export const cU = bas.cU; // U
export const cNp = bas.cNp; // Np
export const cPu = bas.cPu; // Pu
export const cAm = bas.cAm; // Am
export const cCm = bas.cCm; // Cm
export const cBk = bas.cBk; // Bk
export const cCf = bas.cCf; // Cf
export const cEs = bas.cEs; // Es
export const cFm = bas.cFm; // Fm
export const cMd = bas.cMd; // Md
export const cNo = bas.cNo; // No
export const cLr = bas.cLr; // Lr
export const cRf = bas.cRf; // Rf
export const cDb = bas.cDb; // Db
export const cSg = bas.cSg; // Sg
export const cBh = bas.cBh; // Bh
export const cHs = bas.cHs; // Hs
export const cMt = bas.cMt; // Mt
export const cDs = bas.cDs; // Ds
export const cRg = bas.cRg; // Rg
export const cCn = bas.cCn; // Cn
export const cNh = bas.cNh; // Nh
export const cFl = bas.cFl; // Fl
export const cMc = bas.cMc; // Mc
export const cLv = bas.cLv; // Lv
export const cTs = bas.cTs; // Ts
export const cOg = bas.cOg; // Og
// *****************************************************
export const cHydrogen = bas.cHy + bas.cdr + phn.cogen; // Hydrogen
export const cHelium = bas.cHe + phn.clium; // Helium
export const cLithium = wrd.cLit + phn.chium; // Lithium
export const cBeryllium = bas.cBe + bas.cry + phn.cllium; // Beryllium
export const cBoron = bas.cBo + phn.cron; // Boron
export const cCarbon = bas.cCa + bas.crb + bas.con; // Carbon
export const cNitrogen = bas.cNi + bas.ctr + phn.cogen; // Nitrogen
export const cOxygen = bas.cOx + bas.cyg + bas.cen; // Oxygen
export const cFlorine = bas.cFl + phn.corine; // Florine
export const cNeon = bas.cNe + bas.con; // Neon
export const cSodium = bas.cSo + phn.cdium; // Sodium
export const cMagnesium = bas.cMa + bas.cgn + phn.cesium; // Magnesium
export const cAluminum = bas.cAl + bas.cum + phn.cinum; // Aluminum
export const cSilicon = bas.cSi + phn.clic + bas.con; // Silicon
export const cPhosphorus = bas.cPh + bas.cos + bas.cph + bas.cor + bas.cus; // Phosphorus
export const cSulfur = bas.cSu + bas.clf + bas.cur; // Sulfur
export const cClorine = bas.cCl + phn.corine; // Clorine
export const cArgon = gen.cArg + bas.con; // Argon
export const cPotassium = bas.cPo + bas.cta + phn.cssium; // Potassium
export const cCalcium = bas.cCa + bas.clc + phn.cium; // Calcium
export const cScandium = wrd.cScan + phn.cdium; // Scandium
export const cTitanium = bas.cTi + phn.ctan + phn.cium; // Titanium
export const cVanadium = bas.cVa + bas.cna + phn.cdium; // Vanadium
export const cChromium = bas.cCh + bas.cro + phn.cmium; // Chromium
export const cManganese = wrd.cMan + bas.cga + phn.cnes + bas.ce; // Manganese
export const cIron = bas.cIr + bas.con; // Iron
export const cCobalt = bas.cCo + bas.cba + bas.clt; // Cobalt
export const cNickel = bas.cNi + bas.cc + phn.ckel; // Nickel
export const cCopper = bas.cCo + bas.cp + phn.cper; // Copper
export const cZinc = bas.cZi + bas.cnc; // Zinc
export const cGallium = bas.cGa + phn.cllium; // Gallium
export const cGermanium = bas.cGe + phn.crman + phn.cium; // Germanium
export const cArsenic = bas.cAr + bas.cse + phn.cnic; // Arsenic
export const cSelenium = bas.cSe + bas.cle + phn.cnium; // Selenium
export const cBromine = bas.cBr + bas.com + phn.cine; // Bromine
export const cKrypton = bas.cKr + bas.cyp + phn.cton; // Krypton
export const cRubidium = bas.cRu + bas.cbi + phn.cdium; // Rubidium
export const cStrontium = bas.cSt + phn.cron + phn.ctium; // Strontium
export const cYttrium = bas.cYt + phn.ctri + bas.cum; // Yttrium
export const cZirconium = bas.cZi + bas.crc + phn.conium; // Zirconium
export const cNiobium = bas.cNi + bas.cob + phn.cium; // Niobium
export const cMolybdenum = bas.cMo + bas.cly + bas.cbd + bas.cen + bas.cum; // Molybdenum
export const cTechnetium = bas.cTe + bas.cch + bas.cne + phn.ctium; // Technetium
export const cRuthenium = bas.cRu + wrd.cthen + phn.cium; // Ruthenium
export const cRhodium = gen.cRho + phn.cdium; // Rhodium
export const cPalladium = wrd.cPal + bas.cla + phn.cdium; // Palladium
export const cSilver = bas.cSi + bas.cl + phn.cver; // Silver
export const cCadmium = bas.cCa + bas.cdm + phn.cium; // Cadmium
export const cIndium = bas.cIn + phn.cdium; // Indium
export const cTin = bas.cTi + bas.cn; // Tin
export const cAntimony = bas.cAn + bas.cti + phn.cmon + bas.cy; // Antimony
export const cTellurium = bas.cTe + bas.cll + phn.curium; // Tellurium
export const cIodine = bas.cIo + bas.cdi + bas.cne; // Iodine
export const cXenon = bas.cXe + phn.cnon; // Xenon
export const cCesium = bas.cCe + phn.csium; // Cesium
export const cBarium = bas.cBa + phn.crium; // Barium
export const cLanthanum = bas.cLa + wrd.cnth + bas.can + bas.cum; // Lanthanum
export const cCerium = bas.cCe + phn.crium; // Cerium
export const cPraseodymium = bas.cPr + phn.c_ase + phn.codymium; // Praseodymium
export const cNeodymium = bas.cNe + phn.codymium; // Neodymium
export const cPromethium = phn.cPro + phn.cmet + phn.chium; // Promethium
export const cSamarium = bas.cSa + phn.cmar + phn.cium; // Samarium
export const cEuropium = bas.cEu + bas.cro + bas.cpi + bas.cum; // Europium
export const cGadolinium = bas.cGa + bas.cdo + phn.clin + phn.cium; // Gadolinium
export const cTerbium = bas.cTe + phn.crbium; // Terbium
export const cDysprosium = bas.cDy + bas.csp + bas.cro + phn.csium; // Dysprosium
export const cHolmium = bas.cHo + bas.clm + phn.cium; // Holmium
export const cErbium = bas.cEr + phn.cbium; // Erbium
export const cThulium = bas.cTh + bas.cul + phn.cium; // Thulium
export const cYtterbium = bas.cYt + phn.cter + phn.cbium; // Ytterbium
export const cLutetium = bas.cLu + bas.cte + phn.ctium; // Lutetium
export const cHafnium = bas.cHa + bas.cfn + phn.cium; // Hafnium
export const cTantalum = bas.cTa + bas.cn + bas.cta + phn.clum; // Tantalum
export const cTungsten = bas.cTu + bas.cng + phn.cste + bas.cn; // Tungsten
export const cRhenium = bas.cRh + phn.cenium; // Rhenium
export const cOsmium = bas.cOs + phn.cmium; // Osmium
export const cIridium = bas.cIr + phn.cidium; // Iridium
export const cPlatinum = bas.cPl + bas.cat + phn.cinum; // Platinum
export const cGold = bas.cG + wrd.cold; // Gold
export const cMercury = bas.cMe + bas.crc + bas.cur + bas.cy; // Mercury
export const cThallium = bas.cTh + phn.callium; // Thallium
export const cLead = bas.cLe + wrd.c1a1d; // Lead
export const cBismuth = bas.cBi + bas.csm + phn.cuth; // Bismuth
export const cPolonium = bas.cPo + phn.clon + phn.cium; // Polonium
export const cAstatine = bas.cAs + bas.cta + bas.cti + bas.cne; // Astatine
export const cRadon = bas.cRa + bas.cdo + bas.cn; // Radon
export const cFrancium = phn.cFra + phn.cncium; // Francium
export const cRadium = bas.cRa + phn.cdium; // Radium
export const cActinium = wrd.cAct + phn.cinium; // Actinium
export const cThorium = bas.cTh + phn.corium; // Thorium
export const cProtactinium = phn.cPro + bas.cta + phn.cctinium; // Protactinium
export const cUranium = bas.cUr + phn.canium; // Uranium
export const cNeptunium = bas.cNe + bas.cpt + phn.cuni + bas.cum; // Neptunium
export const cPlutonium = bas.cPl + phn.cuto + phn.cnium; // Plutonium
export const cAmericium = bas.cAm + bas.cer + phn.cicium; // Americium
export const cCurium = phn.cCur + phn.cium; // Curium
export const cBerkelium = bas.cBe + bas.crk + phn.celium; // Berkelium
export const cCalifornium = bas.cCa + bas.cli + wrd.cfor + phn.cnium; // Californium
export const cEinsteinium = bas.cEi + bas.cns + bas.cte + phn.cinium; // Einsteinium
export const cFermium = bas.cFe + bas.crm + phn.cium; // Fermium
export const cMendelevium = bas.cMe + bas.cnd + bas.cel + bas.cev + phn.cium; // Mendelevium
export const cNobelium = bas.cNo + bas.cbe + phn.clium; // Nobelium
export const cLawrencium = bas.cLa + bas.cwr + phn.cenc + phn.cium; // Lawrencium
export const cRutherfordium = bas.cRu + phn.cther + wrd.cfor + phn.cdium; // Rutherfordium
export const cDubnium = bas.cDu + bas.cbn + phn.cium; // Dubnium
export const cSeaborgium = wrd.cSea + bas.cbo + bas.crg + phn.cium; // Seaborgium
export const cBohrium = bas.cBo + bas.chr + phn.cium; // Bohrium
export const cHassium = wrd.cHas + phn.csium; // Hassium
export const cMeitnerium = bas.cMe + bas.cit + bas.cne + phn.crium; // Meitnerium
export const cDarmstatium = bas.cDa + bas.crm + bas.cst + phn.cati + bas.cum; // Darmstatium
export const cRoentgenium = bas.cRo + phn.cent + phn.cgen + phn.cium; // Roentgenium
export const cCopernicium = wrd.cCop + phn.cern + phn.cicium; // Copernicium
export const cNihonium = bas.cNi + bas.cho + phn.cnium; // Nihonium
export const cFlerovium = bas.cFl + bas.cer + phn.covium; // Flerovium
export const cMoscovium = bas.cMo + bas.csc + phn.covium; // Moscovium
export const cLivermorium = wrd.cLiver + bas.cmo + phn.crium; // Livermorium
export const cTennessine = num.cTen + phn.cness + phn.cine; // Tennessine
export const cOganesson = bas.cOg + phn.cane + bas.css + bas.con; // Oganesson
